README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
```

Comment:
we can add more details to engage people

Reply:
The problem here is that too many Tetris pieces are the same color. In your review, you might want to suggest that I choose colors which would be easier to see when playing the game.